### ISSUE TITLE
Licenses and Used Content (content)

### ISSUE THREAD

**Issue by liledix4** [liledix4/BvS-Website#64]
A necessary page to avoid adding repetitive license info to other pages. Doing it just once should be good enough.

Besides, this page contains info about reusing website content in other media.

- [ ] #88 
- [ ] #73 
    - [ ] "Can I use anything?"
    - [ ] GitHub repositories of the project
    - [ ] Logos
- [x] Game engines
- [x] #17 
    - [x] Direct links to the texts of licenses
- [x] AI-"generated" content
    - [x] Special message
- [x] Website features

## HTML template for filtered version

- [x] #76 
- [ ] HTML template
    - [ ] #38 
    - [ ] #9
